test/compile.jl

```diff
@@ -32,8 +39,12 @@ try
 
     # use _require_from_serialized to ensure that the test fails if
     # the module doesn't load from the image:
-    println(STDERR, "\nNOTE: The following 'replacing module' warning indicates normal operation:")
```

Comment:
sorry, yes. silence here meant i had moved on to other things, not that i disagreed. i wrote that as breadcrumb, and because i was wondering whether we should avoid broadcasting the reload